### PR TITLE
[Draft][luci-interpreter] draft support for UnidirectionalSequenceLSTM

### PR DESCRIPTION
**Pull request by BalyshevArtem**
This commit adds support for UnidirectionalSequenceLSTM op.

#issue https://github.com/Samsung/ONE/issues/9225

ONE-DCO-1.0-Signed-off-by: Artem Balyshev a.balyshev@partner.samsung.com

### CODE REVIEW

**Review by binarman — COMMENTED** [review]
OMG, this kernel is huge...

File: compiler/luci-interpreter/src/kernels/EvalLSTMHelper.cpp [hunk 1]
```diff
@@ -0,0 +1,758 @@
...
+  // For each batch and cell: compute cell_weight .* cell_state (peephole LSTM)
+  if (use_peephole)
+  {
+    tflite::tensor_utils::VectorBatchVectorCwiseProductAccumulate(cell_to_gate_weights, n_cell,
+                                                                  cell_state, n_batch, gate);
+  }
+  // Do layer normalization (if layer norm LSTM)
+  if (use_layer_norm)
+  {
+    tflite::tensor_utils::MeanStddevNormalization(gate, gate, n_cell, n_batch);
```
[the remark this answers is not in the document]
Comment (binarman) [reply compiler/luci-interpreter/src/kernels/EvalLSTMHelper.cpp:85]: I don't have a good solution in mind (

The only solutions I can think of are:
- copy tensor utils from tensorflow to luci and change aliasing rules for this function
- make temporary buffer for results

**Comment by seanshpark** [comment]
Please check #4396 (and comment https://github.com/Samsung/ONE/pull/4260#issuecomment-705323048)
- `is_variable` is added in compiler but `UnidirectionalSeqeuenceLSTM` is not fixed.

File: compiler/luci-interpreter/src/kernels/EvalLSTMHelper.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (BalyshevArtem) [reply compiler/luci-interpreter/src/kernels/EvalLSTMHelper.cpp]: @binarman, I have updated the draft and choose the first variant: `copy tensor utils from tensorflow to luci and change aliasing rules for this function`

**Pull request closed without merging**